docs/kubernetes/user-guides/snapshots.md

```diff
@@ -1,25 +1,8 @@
-# Kubernetes Snapshots User Guide (Alpha)
+# Kubernetes Snapshots User Guide (Beta)
 
->**Attention:** VolumeSnapshot is an alpha feature. Make sure you have enabled it in Kubernetes API server using `--feature-gates=VolumeSnapshotDataSource=true` flag.
+>**Attention:** Attention: VolumeSnapshot is a Beta feature enabled by default in Kubernetes 1.17+. Attention: VolumeSnapshot is only available in the driver version "master".
 
-### Install Driver with alpha snapshot feature
-
-1. [One-time per project] Create GCP service account for the CSI driver and set required roles
-
-    ```
-    PROJECT=your-project-here                       # GCP project
-    GCE_PD_SA_NAME=my-gce-pd-csi-sa                 # Name of the service account to create
-    GCE_PD_SA_DIR=/my/safe/credentials/directory    # Directory to save the service account key
-    ./deploy/setup-project.sh
-    ```
-
-1. Deploy driver to Kubernetes Cluster
-
-    ```
-    GCE_PD_SA_DIR=/my/safe/credentials/directory    # Directory to get the service account key
-    GCE_PD_DRIVER_VERSION=alpha                     # Driver version to deploy
-    ./deploy/kubernetes/deploy-driver.sh
-    ```
+### Install Driver with beta snapshot feature as described [here] (https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/docs/kubernetes/user-guides/driver-install.md)
```

Comment:
sorry one more nit. I think you can use a relative url here. That way if someone goes to a tagged version of the docs, it'll go to the page from the same tag.

Reply:
Can you please elaborate how the relative url would look like here?

Reply:
The README has some examples: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blame/master/README.md#L90

Reply:
Thanks will take a look